External/SDK/Windows/Windows10SDK.bff

```diff
@@ -71,6 +71,36 @@
                                     + ' /LIBPATH:"$Windows10_SDKBasePath$/Lib/$Windows10_SDKVersion$/ucrt/x64"'
                                     + ' /MACHINE:X64'
 ]
+.Windows10_SDK_ARM64 =
```

Comment:
Is it just the lib paths that differ?

Could be nice to-dupe the common settings here too.